tests/backup/backup_rbac_test.go

```diff
@@ -1244,12 +1606,27 @@ var _ = Describe("{VerifyRBACForAppAdmin}", func() {
 			log.FailOnError(err, "Unable to switch context to source cluster [%s]", SourceClusterName)
 			EndPxBackupTorpedoTest(scheduledAppContexts)
 		}()
+		backupDriver := Inst().Backup
+		bkpScheduleEnumerateReq := &api.BackupScheduleEnumerateRequest{
+			OrgId: orgID,
+		}
+		nonAdminCtx, err := backup.GetNonAdminCtx(appAdminUser, commonPassword)
+		log.FailOnError(err, "failed to fetch user %s ctx", appAdminUser)
+		currentSchedules, err := backupDriver.EnumerateBackupSchedule(nonAdminCtx, bkpScheduleEnumerateReq)
+		log.FailOnError(err, "Getting a list of all schedules")
```

Comment:
dash.VerifySafely?

Reply:
Sorry, had missed it. It's included now.